### PR TITLE
Add directions page

### PR DESCRIPTION
**Pull request by kshepard**

### CODE REVIEW

File: src/app/scripts/cac/control/cac-control-directions-list.js [hunk 1]
```diff
@@ -118,7 +137,7 @@ CAC.Control.DirectionsList = (function ($, Handlebars, UserPreferences, Utils) {
 
     function toggle() {
         if ($container.hasClass('hidden')) {
-            show()
+            show();
```
Comment (CloudNiner) [inline src/app/scripts/cac/control/cac-control-directions-list.js]: Oh yuck. Something must be up with my linter.

File: src/app/scripts/cac/pages/cac-pages-directions.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    };
+
+    function Directions(options) {
+        this.options = $.extend({}, defaults, options);
+    }
+
+    Directions.prototype.initialize = function () {
+        var params = Utils.getUrlParams();
+        if (!_.has(params, 'itineraryIndex')) {
+            console.error('Must specify itineraryIndex URL parameter');
```
Comment (CloudNiner) [inline src/app/scripts/cac/pages/cac-pages-directions.js]: Add a TODO here to show a message in the UI?

**Comment by CloudNiner** [comment]
+1

File: src/app/scripts/cac/control/cac-control-directions-list.js (hunk 1 above)
Comment (kshepard) [inline src/app/scripts/cac/control/cac-control-directions-list.js]: No problem. Related: want to take a look at load_events.py? I just ran the python linter and saw some warnings there.

**Pull request merged**